common/proptest-helpers/src/lib.rs

```diff
@@ -1,6 +1,6 @@
 // Copyright (c) The Diem Core Contributors
 // SPDX-License-Identifier: Apache-2.0
-
+#![allow(clippy::integer_arithmetic)]
```

Comment:
do you really need the other `#![allow(clippy::integer_arithmetic)]` in the crate if you have it in `lib.rs`? Not sure : )

Reply:
you didn't answer the question :P

Reply:
sorry missed this, yes it's not necessary, will remove the line, thanks!